blocks/extra-large-manual-promo-block/themes/news.json

```diff
@@ -28,49 +28,22 @@
 						"justify-content": "center",
 						"line-height": "var(--heading-level-1-line-height)"
 					},
+					"link-hover": {
+						"color": "var(--text-color-subtle)",
+						"opacity": 0.8
```

Comment:
Adding opacity here is making the links too light of a color.

Reply:
@vgalatro, Fixed it in [8a050cf](https://github.com/WPMedia/arc-themes-blocks/pull/1690/commits/8a050cfab3b00863edf7f93c08104a862ad876fe)